src/main/java/com/epam/task/first/Main.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.task.first;
+
+import com.epam.task.first.entities.Array;
+import com.epam.task.first.logic.ArrayLogic;
+import com.epam.task.first.view.ArrayPrinter;
+import com.epam.task.first.view.ConsoleArrayPrinter;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Array array = new Array(33, 41, 13, 1, 5, 18);
+
+        ArrayPrinter printer = new ConsoleArrayPrinter();
+        ArrayLogic arrayLogic = new ArrayLogic();
+
+        System.out.println("Исходный массив:");
```

Comment:
please don't do this. 
Do not use russian, 
you main should not do everything as well